### PR TITLE
Add style guide

### PR DESCRIPTION
**Pull request by anitagraser**

### CODE REVIEW

**Review by yjacolin — APPROVED** [review]

File: source/site/getinvolved/styleguide.rst [hunk 1]
```diff
@@ -0,0 +1,226 @@
+Visual Style Guide
+==================
+
+Version 1.0
+-----------
+
+This is the QGIS Visual Style Guide. This document defines
+standards for the use of the our visual identity. It proposes
```
Comment (DelazJ) [inline source/site/getinvolved/styleguide.rst]: ~~the~~ our

File: source/site/getinvolved/styleguide.rst [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+a modernization of the QGIS brand to make it stronger,
+more visible and more adaptable to a broad range of use
+cases.
+
+The Visual Style Guide is a living document, initiated with
+the goal of developing a consistent message and image
+for QGIS. It is a “living” document because we expect
+it to expand as we cover more use cases where visual
+consistency if important and can be described using
```
Comment (DelazJ) [inline source/site/getinvolved/styleguide.rst]: is

**Comment by anitagraser** [comment]
Thank you both for checking! Should I still wait or should I go ahead with the merge?

**Comment by DelazJ** [comment]
@anitagraser, go ahead, please!

**Pull request merged**